client/components/swipeable/README.md

```diff
@@ -0,0 +1,33 @@
+# Support Info
+
+This component is used to add horizontal swipe controls to a list of elements (pages).
+
+## Example Usage
+
+```js
+import Swipable from 'calypso/components/swipable';
+
+function Pager() {
+	const [ currentPage, setCurrentPage ] = useState( 0 );
+	return ( 
+		<Swipable
+			onPageSelect={ ( index ) => {
+				setCurrentPage( index );
+			} }
+			currentPage
+			pageClassName=""
```

Comment:
Should we provide some value here? If we leave it empty, there's a high chance someone will use it like that in their code.